tensorflow_graphics/projects/pointnet/train.py

```diff
@@ -114,8 +95,7 @@ def evaluate():
   if step % FLAGS.ev_every != 0:
     return evaluate.best_accuracy
   aggregator = tf.keras.metrics.SparseCategoricalAccuracy()
-  for example in ds_test:
-    points, labels = example["points"], example["label"]
+  for points, labels in ds_test:
```

Comment:
should be label, as it's batch of points, and batch of labeL 
(this was my mistake)